MAUI/Autocomplete/UI-Customization.md

```diff
@@ -315,6 +339,156 @@ The following image illustrates the result of the above code:
 
 ![.NET MAUI Autocomplete ItemTemplate](Images/UICustomization/ItemTemplate.png)
 
+### Customize DropDown item text
+
+DropDown items can be customized using the [DropDownItemFontAttributes], [DropDownItemFontFamily], [DropDownItemFontSize] and [DropDownItemTextColor] properties.
```

Comment:
DropDown items can be customized using the [DropDownItemFontAttributes], [DropDownItemFontFamily], [DropDownItemFontSize], and [DropDownItemTextColor] properties.